### ISSUE TITLE
Galaxy S6

### ISSUE THREAD

**Issue by julioyg** [commonsguy/cwac-cam2#143]
I got an issue trying to take a picture with a galaxy S6: press the capture button, it turns grey but the photo is not being taken till the focus changes dramatically like covering the camera. it works if I add that device as problematic `isProblematicDeviceOnNewCameraApi()`

**Comment by commonsguy** [comment]
Can you provide me with the specific `Build.PRODUCT` and `Build.MANUFACTURER` values for this device? The latter is probably `samsung`. Thanks!

**Comment by julioyg** [comment]
sorry about the time I took to answer. I added these conditions to you method `isProblematicDeviceOnNewCameraApi()`:

```
    if ("samsung".equals(Build.MANUFACTURER) &&
            "zerofltexx".equals(Build.PRODUCT)) {
      return(true);
    }
```

that's what I got from the device

**Comment by julioyg** [comment]
wrong button

**Comment by commonsguy** [comment]
OK, that's the SM-G920F. I'll see if I can get my hands on one someday.

In the meantime, I'll look to add your values to `isProblematicDeviceOnNewCameraApi()` for the next update.

Thanks!

**Comment by commonsguy** [comment]
Addressed in v0.5.0.
